src/test/tests/databases/export_db.py

```diff
@@ -126,15 +134,107 @@ def VTK_check_binary(filename):
+"STL":          ("export_STL.stl",          "*.stl",         "Subset",      "STL_mesh"),
+"Silo":         ("export_Silo.silo",        "wg_Silo.silo",  "Pseudocolor", "u"),
+"Tecplot":      ("export_Tecplot.tec",      "*.tec",         "Pseudocolor", "u"),
+"VTK":          ("export_VTK.visit",        "wg_VTK.visit",  "Pseudocolor", "u"),
+"WavefrontOBJ": ("export_WavefrontOBJ.obj", "*.obj",         "Subset",      "OBJMesh"),
+"XYZ":          ("export_XYZ.xyz",          "*.xyz",         "Subset",      "mesh"),
+"Xmdv":         ("export_Xmdv.visit",       "wg_Xmdv.visit", "Pseudocolor", "u")
+}
+
+    keys = sorted(formats.keys())
+    test_name_base="export_db_%d"%testNum
+   
+    # do all the exports
+    for f in keys:
+        export_name="%s_%s"%("export" if testNum==1 else "wg", f)
+        e.db_type = f
+        e.db_type_fullname = f + "_1.0"
+        e.filename = export_name
+        ExportDatabase(e)
+        time.sleep(1)
+
+    # now attempt to read the exported files
+    if testNum == 2:
+        files = sorted(fnmatch.filter(os.listdir(e.dirname), "wg_*"))
+    for f in keys:
+        # Add the exported data in window 2.
+        AddWindow()
+
+        pattern = formats[f][0] if testNum ==1 else formats[f][1]
+        filelist=""
+        opendbs = []
+
+        if pattern[0] == '*':
+            # Read all of the filenames
+            ext = pattern[2:]
+            for datafile in files:
+                if datafile[-len(ext):] == ext:
+                    if OpenDatabase(datafile):
+                        AddPlot(formats[f][2], formats[f][3])
+                        opendbs = opendbs + [datafile]
+                        filelist = filelist + datafile + "\n"
+            DrawPlots()
+        else:
+            if OpenDatabase(pjoin(e.dirname, pattern)):
+                md = GetMetaData(pattern)
+                AddPlot(formats[f][2], formats[f][3])
+                DrawPlots()
+                if testNum == 2:
+                    opendbs = opendbs + [pattern]
+                    filelist = filelist + pattern + "\n"
+            else:
+                if testNum == 1:
+                    files = os.listdir(e.dirname)
+                    files.sort()
+                    s = ""
+                    for fn in files:
+                        if pattern in fn:
+                            s = fn + "\n"
+                    TestText("files", s)
+                else:
+                    filelist = "ERROR: " + "\n".join(os.listdir("."))
+
+        t = CreateAnnotationObject("Text2D")
+        t.text = f
+        t.position = (0.01, 0.91)
+        t.height = 0.07
+        SetView3D(v)
+        SetAnnotationAttributes(a)
+        Test("%s_%02d"%(test_name_base, count))
+        if testNum == 2:
+            TestText("%s_%02dfn"%(test_name_base, count),filelist)
+        count=count+1
+
+        # Clean up window 2
+        DeleteAllPlots()
+        #if testNum == 1:
```

Comment:
Should this be commented out?

Reply:
The CloseDatabase calls were commented out in original code.

Reply:
Does it fail with them in?

Reply:
I can try.

Reply:
There are other places in the file where CloseDatabase calls are commented out, should I uncomment them all?

Reply:
A long while back, I recall having to comment out calls to `CloseDatabase()` in a number of places due to funky behavior. So, it might just still fail. If it does, just leave them commented out but maybe add a new ticket to resolve them.

Reply:
Uncommenting all CloseDatabase calls causes a problem in parallel. I will leave them all commented for now and issue new ticket as suggested.